tensorflow_probability/python/experimental/mcmc/weighted_resampling_test.py

```diff
@@ -299,6 +299,46 @@ def resample_with_target_distribution(self):
         tf.reduce_sum(tf.nn.softmax(new_log_weights) * new_particles),
         30., atol=1.)
 
+  def test_with_target_distribution_dim_one(self):
+    particles = np.linspace(0., 500., num=2500, dtype=np.float32)
+    stacked_particles = np.stack([particles, particles], axis=0)
```

Comment:
It may be worth making the two different batch dimensions contain different particles -- e.g., something like:
```
stacked_particles = np.stack([
    np.linspace(0., 500., num=2500, dtype=np.float32),
    np.linspace(0.17, 433., num=2500, dtype=np.float32),
], axis=0)
```